src/ShalekKavy.Api/Services/IBeverageRepository.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using ShalekKavy.Api.Models.Enums;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+namespace ShalekKavy.Api.Services
+{
+    public interface IBeverageRepository
+    {
+        public IQueryable<Beverage> GetAllBeverages();
```

Comment:
I am not sure why you would need to expose IQueruable from the Repo. The idea of the repo pattern is to do the DB stuff within the repo and not exposing the db outside of it.

Reply:
Removed